### DISCUSSION TITLE
No more gc readings from Liverview via webfollower?

### DISCUSSION THREAD

**Discussion by RonaldMigo** [NightscoutFoundation/xDrip#3230] [Ideas]
Since 2 days the Xdrip app running on my son's phone is not getting  data anymore from libreview. It now just returns errors like this: 
![image](https://github.com/NightscoutFoundation/xDrip/assets/22676231/b803f8a3-61da-4edc-9c91-b9684cc1e47f)

Anyone else having issues pulling data from libreview??

cheers

**Comment by PhilTU** [comment]
In general when I see error messages with "terms of use" or "accept" or similar terms the solution is to log out of the official LibreLinkUp app, log back in and accept the periodic terms & conditions or privacy update notices.

**Reply by RonaldMigo** [reply]
Thanks. I created a second follower account and now it is working again. You basically have to remove the follower from the master libre app that is used for scanning the sensor. Then add the follower again. A second follower account (what I did) shouldn’t be necessary.

**Reply by Navid200** [reply]
What is the sensor?  Is it Libre 3?

**Reply by RonaldMigo** [reply]
Yes it is a libre 3 sensor.
